docs/Modifying_Clearwater_settings.md

```diff
@@ -27,9 +27,9 @@ Some of the more complex sprout-specific configuration is stored in JSON files
 
 To change one of these files:
 
-* Edit the file on *one* of your sprout nodes in each site.
-* Run one of `sudo cw-upload_{scscf|bgcf|enum}_json` depending on which file you modified.
-* The change will be automatically propagated around the site (by Clearwater's [automatic configuration sharing](Automatic_Clustering_Config_Sharing.md) functionality) and will start being used.
+*   Run `cw-config download {scscf|bgcf|enum}_json` on *one* node in each site. This will download the current version of `{scscf|bgcf|enum}_json` to `~/clearwater-config-manager/[username]/{scscf|bgcf|enum}.json`.
```

Comment:
Similarly - I wouldn't reference the username part. Just say that the command tells you the file to edit.